app/web_conn.go

```diff
@@ -749,6 +749,11 @@ func (wc *WebConn) shouldSendEvent(msg *model.WebSocketEvent) bool {
 		return wc.GetConnectionID() == msg.GetBroadcast().ConnectionId
 	}
 
+	// if the connection is omitted don't send the message
+	if msg.GetBroadcast().OmitConnection != "" {
```

Comment:
Just to confirm, are you okay with this filter set as a global override? At a global level, this will override any broadcast to channel or team scoped events. I thought you wanted this override as part of any event?

For example, if there's a channel scoped event, but we don't want to send it to a given connectionId, we can't do that here. It will always be a global broadcast, except a given connectionId.

Are you okay with that?

Reply:
@agnivade for my specific use case that's OK, but yes, maybe it should support scoped events as well.

Reply:
Understood. @streamer45 - what do you think? I think `OmitUsers` should support channel and team scope as well. It's just more intuitive to think that way. Otherwise, I have to always remember the priority order.

Reply:
Personally I think we should return early only if the connection id matches with the omit value but otherwise go ahead with the other checks.

Reply:
Ok, so you are good with the current logic.

Reply:
I'm a bit confused, if we return only when false and otherwise pass through, then it will support the other flags, no?

Reply:
Current logic is returning early regardless (as long as `OmitConnectionId` is non-empty) which feels unexpected. I am suggesting to return early (`false`) only if the ids match.

Reply:
Ah ok, so you want it to fall through and check other scopes. Yes, I agree. And I think we should do it for `OmitUsers` as well to keep the logic consistent. i.e. for `deny` checks, deny only if it matches, else go ahead with other checks.

Reply:
Sounds good although changing existing behaviour may need some extra care.

Reply:
Yeah we just need to confirm `OmitUsers` are always set alone without any other overrides.

Reply:
Oh wait, it's already doing what I said. So all good. Nothing to change :+1: